packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -1454,16 +1454,21 @@ class RenderClipRect extends _RenderCustomClip<Rect> {
 
   @override
   void paint(PaintingContext context, Offset offset) {
+    _updateClip();
```

Comment:
Do we need to do this unconditionally now? Seems like we only need it if we actually clip?

Reply:
I remember doing this intentionally, but do not remember why. 🙃 
I'll move it and see what shakes out.

Reply:
Pre-holiday code in a post-holiday world is perplexing.

Reply:
So it looks like two golden files are affected by moving this, not clear yet why. But I also noticed that `_clip` is set by `updateClip`. `_clip` is used for hit testing in all of these, so it would definitely cause an issue if we did not set it.. let me rework this a bit to account for the hit testing (it does not have test coverage either) and find out why the goldens are affected. Thanks!

Reply:
Isn't hit testing just updating the clip if necessary?